draft-kazuho-httpbis-priority.md

```diff
@@ -307,6 +307,45 @@ receipt, a client that supports header-based prioritization MUST close the
 connection with a protocol error. Non-supporting clients will ignore this
 extension element (see {{!RFC7540}}, Section 5.5).
 
+# Security Considerations
+
+## Fairness and Coalescing Intermediaries {#fairness}
+
+When an intermediary coalesces HTTP requests coming from multiple clients into
+one HTTP/2 or HTTP/3 connection going to the backend server, requests that
+originate from one client might have higher precedence than those coming from
+others.
+
+It is sometimes beneficial for the server running behind to obey to the value of
```

Comment:
```suggestion
It is sometimes beneficial for the server running behind an intermediary to obey to the value of
```

Just a little suggestion for readability. I know this is in a section that discusses intermediaries but on two separate readings the sentence construct made me pause to think